src/clarity/iconography/demo/icon-single-select.html

```diff
@@ -0,0 +1 @@
+<clr-icon [attr.shape]="shape"></clr-icon><span class="icon-shape-name">{{shape}}</span>
```

Comment:
Just a remark, not a requested change: when your template is literally one line, you can just inline it in the Typescript file. It's easier to read for everyone. 😛